BootloaderCorePkg/Library/MpInitLib/Ia32/MpInitLib.c

```diff
@@ -96,6 +97,11 @@ CpuInit (
     SmmRebase (Index, ApicId, 0);
   }
 
+  PlatformCpuInitHook = (PLATFORM_CPU_INIT_HOOK)(UINTN)PcdGet32 (PcdPlatformCpuInitHook);
+  if (PlatformCpuInitHook != NULL) {
+    PlatformCpuInitHook ();
```

Comment:
I think it is better to pass in the Index so that it be used to identify this CPU related info.

Reply:
OK, will add it.